run/test/specs/group_disappearing_messages_link.spec.ts

```diff
@@ -25,7 +25,7 @@ async function disappearingLinkMessageGroup(platform: SupportedPlatformsType) {
   ]);
   await createGroup(platform, device1, userA, device2, userB, device3, userC, testGroupName);
   await setDisappearingMessage(platform, device1, ['Group', timerType, time]);
-  // await device1.navigateBack(platform);
+  // await device1.navigateBack();
```

Comment:
not sure why you keep all of those commented navigateBack

Reply:
For fun